JASP-Engine/JASP/R/commonerrorcheck.R

```diff
@@ -91,6 +91,27 @@
       message <- paste0(openingMsg, '<ul>', message)
     }
   }
+
+  # See if we should add the variables causing the error (so far only for perfect correlations).
+  if (type == "varCovData" && ! is.null(args$variables)) {
+    nPairs <- length(args$variables$var1) # Could take either var1 or var2 as length is identical
+    if (nPairs > 0) {
```

Comment:
Question: can `nPairs` even have length 0? Because you already check if `args$variables` is`NULL`.

Reply:
In the way it was programmed previously it could. But I can see how this is confusing. Therefore, I changed the code in the .checkForPerfectCorrelations function.